insights/config/specs.py

```diff
@@ -8,8 +8,8 @@
 specs listed here are collected.
 """
 
-from insights.config import SimpleFileSpec, PatternSpec, CommandSpec, format_rpm, json_format, First, All, NoneGroup, format_rpm_old
```

Comment:
I am not sure if it is correct to remove the json_format. If it is correct,  I think you should also remove the definition of josn_format.

Reply:
@zhangyi733 - I prefer to remove the definition of `josn_format`, how do you think?

Reply:
@shzhou12 and @xiangce - Since no functions involve `json_format`, I agree to remove it.